plugins/BEdita/Core/src/Model/Action/SaveEntityAction.php

```diff
@@ -56,15 +56,15 @@ public function execute(array $data = [])
         if ($success === false) {
             $errors = $entity->getErrors();
             if (!empty($errors)) {
-                $this->log('Entity save failed', 'debug', compact('entity'));
+                $this->log(sprintf('Entity save errors: %s', print_r($errors, true)), 'warning');
```

Comment:
Probably `json_encode` is better. Services that ingest logs often have parsers for JSON (even if mixed with other plain test), but not for PHP's `print_r` format.